### PR TITLE
solution

### PR DESCRIPTION
**Pull request by tanyakovchuk**
[DEMO LINK](https://taya294.github.io/react_movies-list/)

### CODE REVIEW

**Review by Grygoriy-Shytikov — APPROVED** [review]
Looks good 👍

File: src/components/MoviesList/MoviesList.js [hunk 1]
```diff
@@ -1,6 +1,22 @@
...
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+export const MoviesList = ({ movies }) => (
+  <>
+    {movies.map(movie => (
+      <div className="card" key={movie.imdbId}>
+        <MovieCard {...movie} />
+      </div>
```
Comment (VinnieJ-2k20) [inline src/components/MoviesList/MoviesList.js:11]: Move this `div` inside `MovieCard` component. Use fragments only when necessary.
```suggestion
      <MovieCard {...movie} key={movie.imdbId} />
```

**Review by VinnieJ-2k20 — CHANGES_REQUESTED** [review]
Nice solution. Just one small thing to fix.

File: src/components/MovieCard/MovieCard.js [hunk 2]
```diff
@@ -1,6 +1,44 @@
 import React from 'react';
 import './MovieCard.scss';
+import PropTypes from 'prop-types';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ title, description, imgUrl, imdbUrl }) => (
+  <>
```
Comment (aliceblunt8) [inline src/components/MovieCard/MovieCard.js:6]: remove fragment, it's unnecessary here

**Review by aliceblunt8 — APPROVED** [review]

**Pull request closed without merging**